archive/qt_installer_framework.ksy

```diff
@@ -0,0 +1,225 @@
+meta:
+  id: qt_installer_framework
+  title: Qt Installer Framework archive
+  application: Qt Installer Framework
+  file-extension: exe
+  endian: le
+  encoding: utf-8
+  license: GPL-3.0  # With exceptions, see https://github.com/qtproject/installer-framework/blob/master/LICENSE.GPL3-EXCEPT
+  xref:
+    wikidata: Q201904
+
+doc: |
+  Qt installer framework is a set of libs to make SFX installers. Installers usually contain 7zip-compressed archives. Obviously, Qt installer itself is built using this framework.
+  
+  Warning 1: KSC has a bug. It makes the computed values be int32_t. Of course their type should be either explicitly specified by a programmer or derived automatically. The workaroind is to just replace all int32_t to int64_t in sources.
+  Warning 2: don't use this spec on Linux against Qt distribution with overcommit enabled unless you have lot of RAM (> 12 GiB). There is a severe memory leak somewhere (currently I have no idea where exactly). The leak is present in both C++ and python-compiled code. In python even if I have patched the generated source to explicitly free all the `bytes`, `BytesIO`s and `KaitaiStream` objects the leak is still present. At least it is neither in `bytes` nor in `BytesIO`. In C++ I have not patched anything but used move semantics to free the stuff since std::unique_ptr is used. The leak is still present. IDK where it is and how to fix it.
```

Comment:
Can you please wrap the long lines to 80 characters, as the [KSY Style Guide](https://doc.kaitai.io/ksy_style_guide.html#doc) recommends?

> Lines should be wrapped to be 80 columns long. If it doesn’t fit into single line after wrapping, then it’s a multi-line docstring, so use proper multi-line syntax.

Reply:
I am against manual wrapping. Instead wrapping must be done by a text editor depending on window size.